samples/order-management-system/README.md

```diff
@@ -0,0 +1,156 @@
+# Order Management Scenario
+
+## Scenario
+
+This scenario is about a order management system. It consists of two parties. Those are a customer and an order management system(OMS).
+In our sample we are hosting our OMS in  WSO2 Application Server. Customer would interact with OMS to place orders, view orders, make payments and view payments.
+As the order management system we will be using a simple java web service which we will be hosting in WSO2 AS.
+This is a SOAP service which is used to store all the order details in an in-memory table. To simulate customer operations, we will be using a simple HTTP client tool like curl.
+Because our backend(OMS) is a SOAP based backend we will have to use some sort of Rest to SOAP converter. This will be done by Enterprise Integrator.
+
+## Requirements
+
+###Setting up servers
```

Comment:
Does number of # signs have a meaning? 
If not, shall we have a consistent number of # signs to denote a header?

Reply:
Yes It has. It will change the font size.